CVE-2020-15906/data/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+ web:
+   image: vulhub/tikiwiki:21.1
```

Comment:
Please download the Docker image and keep it in the data folder. Install it from the file.